frictionless/formats/sql/adapter.py

```diff
@@ -122,6 +122,7 @@ def write_schema(
         table_name: str,
         force: bool = False,
         with_metadata: bool = False,
+        without_constraints: bool = False,
```

Comment:
nitpick : I think `ignore_constraints` would be a little bit more explicit

Reply:
Thanks! Renamed to `ignore_constraints`